src/runtime/pay-client-test.js

```diff
@@ -407,7 +454,7 @@ describes.realWin('PayClientBindingPayjs', {}, env => {
     payClientStubs = {
       create: sandbox
         .stub(PayClientBindingPayjs.prototype, 'createClient_')
-        .callsFake((options, handler) => {
+        .callsFake((options, googleTransactionId, handler) => {
           responseHandler = handler;
```

Comment:
Its probably not an issue (and not caused by your development if it is) but should we be using options or googleTransactionId here?  If we have some other method of using it, maybe it would be cleaner to do so here.

Reply:
I didn't set it in options because it needs to be passed back to pay-flow, so I set it as a member of PayClient.